test/edu/ucsb/nceas/metacattest/EventLogIT.java

```diff
@@ -24,23 +30,29 @@ public class EventLogTest extends MCTestCase {
                     + "useragent-useragent-useragent-useragent-useragent-useragent-useragent-"
                     + "useragent-useragent-useragent-useragent-useragent-useragent-useragent-"
                     + "useragent-useragent-useragent-useragent-useragent-useragent-useragent-12";
-    protected void setUp() throws Exception {
-        super.setUp();
-        DBConnectionPool pool = DBConnectionPool.getInstance();
+
+    /**
+     * Establish the property service
+     */
+    @Before
+    public void setUp() throws Exception {
+        LeanTestUtils.initializePropertyService(LeanTestUtils.PropertiesMode.LIVE_TEST);
```

Comment:
this can be `LeanTestUtils.PropertiesMode.UNIT_TEST`

(the LIVE part refers to which version of properties to use - so in this case we can use test properties and mock values - see below)